src/contract_client/sent_transaction.ts

```diff
@@ -72,6 +72,17 @@ export class SentTransaction<T> {
 
   private send = async (): Promise<this> => {
     const timeoutInSeconds = this.assembled.options.timeoutInSeconds ?? DEFAULT_TIMEOUT
+    const timeoutTimestamp =
+      Math.floor(Date.now() / 1000) + timeoutInSeconds + 1;
+    this.assembled.built = TransactionBuilder.cloneFrom(this.assembled.built!, {
+      fee: this.assembled.built!.fee,
+      timebounds: {
+        minTime: 0,
+        maxTime: timeoutTimestamp,
+      },
+    })
```

Comment:
Can you simplify this with `.setTimeout(timeoutInSeconds + 1)` which _should_ be equivalent?

Reply:
oh wat okay that's weird - good call, thanks

Reply:
ah yeah of course, because of this check: https://stellar.github.io/js-stellar-sdk/js-stellar-base_src_transaction_builder.js.html#line281 and the timebound is set in the cloned version on construction... you could, then, do something like `{ timebounds: undefined }` to overwrite the existing one, then `.setTimeout()`

Reply:
done